security.md

```diff
@@ -37,3 +37,90 @@ We are working to identify and coordinate security efforts across the Jupyter co
 The [Jupyter Security](https://github.com/jupyter/security) GitHub repo has information how to participate and contribute.
 For discussion, please use the special Discourse [security topic](https://discourse.jupyter.org/c/special-topics/security/48) on the Jupyter Discourse server.
 
+----
+
+Below is a copy of the vulnerability handling procedure that can be found on the [security repository](https://github.com/jupyter/security).
+
+# Jupyter Vulnerability Handling Process
+
+This document summarizes and proposes guidelines for handling vulnerabilities reported in Jupyter projects. 
+
+Security issues and vulnerabilities have expectations and processes that are differ from typical open source practices:
```

Comment:
The way this is phrased makes it difficult to tell if the bullet points are a description of "typical open source pratices" or "the different ones used for security". I know which are you are talking about, but it may be more sense to talk about these as tenets of the security handling process.